message-queue-lib/src/cmr/message_queue/queue/sqs.clj

```diff
@@ -244,16 +294,75 @@
 
 (defn- -get-queue-url
   "Returns the queue url for the given queue name."
-  [sqs-client queue-name]
+  [sqs-client normalized-lookup queue-name]
```

Comment:
Looks like `normalized-lookup` was not needed after your debugging was finished. Can you remove it?

Reply:
Will do.

Reply:
Done.